src/slash/open.ts

```diff
@@ -261,6 +263,6 @@ export class FriendCodeModalHandler implements MessageModalSubmitHandler {
 			});
 			return;
 		}
-		await registerParticipant(interaction, tournament, friendCode);
+		await registerParticipant(interaction, tournament, friendCode, interaction.fields.getTextInputValue("ign"));
```

Comment:
I wouldn't mind having an explicit `const ign = interaction.fields.getTextInputValue("ign")` next to friend code just to explicitly list all the parameters we're loading in one place, but that's entirely up to taste.